Sources/MapboxCoreNavigation/ReplayLocationManager.swift

```diff
@@ -7,15 +7,14 @@ import CoreLocation
  adjusted by interval between locations.
  */
 open class ReplayLocationManager: NavigationLocationManager {
+    
+    // MARK: Simulation Controls
```

Comment:
```suggestion
    // MARK: Specifying What to Simulate
```

Reply:
Under `Specifying What to Simulate` mark I would expect methods to set coordinates/routes or something like this. But here we have not only 'what' but also 'how' to simulate. So I would suggest something wider like `Specifying Simulation`